src/components/TodoTop.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+const TodoTopStyle=styled.div`
+
+    .todoTopContainer{
+        display:flex;
+        flex-direction:row;
+        justify-content:space-evenly;
+        padding:20px;
+        align-items:center;
+    }
+    .todoTopTitle{
+        width:230px;
+        height:60px;
+        font-size:50px;
+        font-weight:700;
+
+    }
+    .todoTopDark{
+        width:40px;
+        height:40px;
+        color:white;
+    }
+`;
+
+function TodoTop(){
+    const [darkMode, setDarkMode] = useState(false);
+
+  const handleDarkMode = () => {
+    setDarkMode(!darkMode);
+    if (!darkMode) {
+      document.body.style.backgroundColor = "#282c34";
+      document.body.style.color = "#e9ecef";
+    } else {
+      document.body.style.backgroundColor = "#e9ecef";
+      document.body.style.color = "black";
+    }
+  };
```

Comment:
setState는 해당 라인의 코드가 읽힌 후 바로 작동하지 않기 때문에 위와 같이 작성해주신 것 같아요!
물론 개발자들은  setState가 동기적으로 작동하지않는것을 알고있겠지만, 아래와 같이 순서를 바꿔 작성한다면 조금 더 가독성이 좋고, 혹시 모를 sideEffect를 방지하는 효과도 있지않을까 싶습니다!
(setDarkMode를 함수 제일 아래로 내렸습니다!)
```suggestion
  const handleDarkMode = () => {
    if (!darkMode) {
      document.body.style.backgroundColor = "#282c34";
      document.body.style.color = "#e9ecef";
    } else {
      document.body.style.backgroundColor = "#e9ecef";
      document.body.style.color = "black";
    }
    setDarkMode(!darkMode);
  };
```

Reply:
감사합니다